### ISSUE TITLE
Eclipse SARL 0.11.0 on macOS 10.15.6 Java 11 issue

### ISSUE THREAD

**Issue by ThomasFarrenkopf** [sarl/sarl#1022]
Dear all,

we have issues running the latest version on a mac os computer. We are running OpenJDK 64-Bit Server VM AdoptOpenJDK (build 11.0.8+10, mixed mode) and when we try to open it, we get `LSOpenURLsWithRole() failed for the application /Applications/Eclipse SARL.app with error -10810.`

How can we resolve this?

![image](https://user-images.githubusercontent.com/22127913/89624735-f8422580-d896-11ea-880d-4ea6cf9ae39d.png)

The Screenshot just shows the error message "_The "Eclipse SARL" program cannot be opened_".

Yours, Thomas

**Comment by gallandarakhneorg** [comment]
Thank you for the issue report. We are currently working on the support of Java 11 that was not complete yet. A part of the answer is related to Issue #1012.

**Comment by gallandarakhneorg** [comment]
Error code on Mac box is:
```
Error -10810 is a Launch Services result code indicating an unknown error. One cause of this error is that the Mac® OS X process table is full. When the process table is full, new (not currently running) applications cannot be opened until another running application ends.
```

**Comment by gallandarakhneorg** [comment]
Dear @ThomasFarrenkopf, according to the explaination I have found on the error code, did you try to open SARL environment without any other application opened before (to be sure the maximum number of processes is not reached)?

**Comment by gallandarakhneorg** [comment]
Anyway, did you already install and start a standard Eclipse product on your MacOS box? Indeed the SARL product is based on Eclipse. If a standard Eclipse, e.g. the Eclipse for Java Developpers version, cannot be started, then there is no chance to have the SARL environment started.

**Comment by ThomasFarrenkopf** [comment]
Hi @gallandarakhneorg, I'am using Eclipse for Java Developers. This runs without any issues.

Students from my course have exactly the same problems. A workaround is using SARL on a virtual machine (Windows). The problem also exists with your nightly build.

It does not seem to matter whether Java 8 or Java 11 is used.

**Comment by gallandarakhneorg** [comment]
Dear @ThomasFarrenkopf; thank you for your inputs.

From the folder `Eclipse.app/Contents/Eclipse`, are you able to run on the following command line with success?

```
 java -jar plugins/org.eclipse.equinox.launcher_$VERSION.jar
```

where `$VERSION` should be replaced by the version of the library int the folder.

**Comment by ThomasFarrenkopf** [comment]
Hi @gallandarakhneorg,

I found the reason. Latest build run. 

The message which appears at the user interface is exactly the same as posted on 7 Aug.

But the system.log entry is different:
(application.io.sarl.lang.product.72020573.72020798[78958]): removing service since it exited with consistent failure - OS_REASON_EXEC | Gatekeeper policy blocked execution

So... the Gatekeeper is blocking the Application, because it always crashed at startup. Strange.

What I did is just remove the quarantine entry for the application by using the following command:

sudo xattr -rd com.apple.quarantine Eclipse.app

Now it runs ;-)

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/22127913/100582023-dc9cd800-32e8-11eb-9835-225129ff04f0.png">

Thanks a lot!

**Comment by gallandarakhneorg** [comment]
Dear @ThomasFarrenkopf.
Thank you for the update and the good news.

I just installed Catalina on a VirtualBox to test SARL, and did not reproduce the issue. Your message confirm that I did a correct installation of the MacOS environment, that is new for me :-)

Nevertheless, I will add a short message in the FAQ of SARL to help other MacOS users.
Because you have faced the problem and found the answer, I may propose to you to be pushed as a contributor to the SARL documentation for this new FAQ entry. I'm able to push on the Git for you. Are you agree?
